manim/mobject/graph.py

```
@@ -338,6 +380,68 @@ def update_edge(e, u=u, v=v):
             update_edge(edge)
             edge.add_updater(update_edge)
 
+    def _tree_layout(
+        self,
+        G: nx.classes.graph.Graph,
+        root_vertex: Union[Hashable, None],
+        scale: float,
+    ) -> dict:
+        result = {root_vertex: np.array([0, 0, 0])}
+
+        if not nx.is_tree(G):
+            print(
```

I'd rather raise an exception: tree layouts are intended for trees.